base/libgit2/libgit2.jl

```diff
@@ -212,7 +268,16 @@ function branch(repo::GitRepo)
     end
 end
 
-""" git checkout [-b|-B] <branch> [<start-point>] [--track <remote>/<branch>] """
+"""
+    branch!(repo::GitRepo, branch_name::AbstractString, commit::AbstractString="", track::AbstractString="", force::Bool=false, set_head::Bool=true)
+
+Equivalent to `git checkout [-b|-B] <branch> [<start-point>] [--track <remote>/<branch>]`.
```

Comment:
Would be nice to map the function arguments to the appropriate pieces in this Git command